d2cli/main.go

```diff
@@ -260,6 +261,11 @@ func Run(ctx context.Context, ms *xmain.State) (err error) {
 			centerFlag = nil
 		}
 	}
+	if ms.Env.Getenv("D2_BOARD") == "" {
```

Comment:
Does this need an environment variable counterpart? It feels highly specific to each diagram, not something I'd see anyone wanting to preserve in environment.